packages/scaffolding-cli/src/domain/config/static.config.json

```diff
@@ -95,22 +99,38 @@
     ]
   },
   "netcore": {
-    "git_repo": "https://github.com/aspnet/samples.git",
-    "git_ref": "492e3e586a9e20a75dd55537e2acb9d5c011d0c0",
+    "git_repo": "https://github.com/amido/stacks-dotnet.git",
+    "git_ref": "a19e47e0a0b5a0f8c171a2ee2db4318967eb42c9",
```

Comment:
Can we reference the tag instead of the commit SHA?

Reply:
yes which one though :)